### PR TITLE
Allow to set the client is_first_party property

### LINKED ISSUE #229: Set method for isFirstParty is missing in Client.java
The set method for **isFirstParty** is missing in Client.java

https://github.com/auth0/auth0-java/blob/master/src/main/java/com/auth0/json/mgmt/client/Client.java

### PR DESCRIPTION
**Pull request by lbalmaceda**
### Changes

The property should be modifiable, as per the updated docs.

### References

Closes #229 

### Testing

- [x] This change adds test coverage
- [x] This change has been tested on the latest version of the platform/language or why not

### Checklist

- [x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
- [x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
- [x] All existing and new tests complete without errors

### CODE REVIEW

File: src/main/java/com/auth0/json/mgmt/client/Client.java [hunk 1]
```diff
@@ -205,6 +205,14 @@ public Boolean isFirstParty() {
         return isFirstParty;
     }
 
+    /**
+     * Setter for whether this application is a first party client or not.
+     */
+    @JsonProperty("is_first_party")
+    public void setFirstParty(Boolean isFirstParty) {
```
Comment (damieng) [inline src/main/java/com/auth0/json/mgmt/client/Client.java]: This should be really named setIsFirstParty or we're going to run into problems with code generation.

Comment (lbalmaceda) [reply src/main/java/com/auth0/json/mgmt/client/Client.java]: @damieng I checked that and the rest of the methods in all the classes (even those other than Client) are not using "set IS x" but "set x".

**Comment by lbalmaceda** [comment]
@damieng Will need a force merge please, as codecov is not reporting.

image

**Comment by damieng** [comment]
I've made codecov no longer be required for push - it's not reliable enough any more.

**Pull request merged**